ERP-BACKEND/app/core/inventory/dependency_graph.py

```
@@ -0,0 +1,162 @@
+        
+        def find_cycles(node: str, path: List[str]) -> None:
+            visited.add(node)
+            path.append(node)
+            
+            for neighbor in self.get_module_dependencies(node):
+                if neighbor not in visited:
+                    find_cycles(neighbor, path.copy())
+                elif neighbor in path:
+                    # Found a cycle
+                    cycle_start = path.index(neighbor)
+                    cycle = path[cycle_start:] + [neighbor]
```

Ruff (0.16.1)

[warning] Consider `[*path[cycle_start:], neighbor]` instead of concatenation

Replace with `[*path[cycle_start:], neighbor]`

(RUF005)